tests/specs/graph/jsr/FastCheck_ClassPropertiesWithTsPrivate.txt

```diff
@@ -68,12 +70,14 @@ Fast check https://jsr.io/@scope/a/1.0.0/mod.ts:
   }
   export class WithTsPrivate extends Base {
     declare private regular: any;
-    private override withOverride: any;
+    declare private definiteShouldGoAway: any;
```

Comment:
Nit: We should probably rename this property, because the name doesn't match anymore.

Reply:
This is the output with it going away. The input has the `!`